app/models/blacklisted_event.rb

```diff
@@ -34,20 +34,6 @@ def self.current_userid
   end
 
   def queue_sync_blacklisted_event_names
```

Comment:
Super minor, but this method name no longer makes sense (sort of)

Reply:
+1 I'll rename it.